### DISCUSSION TITLE
can support chinese?

### DISCUSSION THREAD

**Discussion by lkzwc** [TriliumNext/Trilium#3659] [Ideas]
can support chinese?can support chinese?can support chinese?

**Comment by zadam** [comment]
Hi, see https://github.com/zadam/trilium/issues/3603

**Comment by lkzwc** [comment]
The best product, no one

**Comment by Nriver** [comment]
I made a Chinese translation. Just released the latest 0.59.1
Here it is. https://github.com/Nriver/trilium-translation/blob/main/README_CN.md
